### PR TITLE
docs(widget-chat): Widget docs

### PR DESCRIPTION
**Pull request by bzang**

### CODE REVIEW

File: packages/widget-message-meet/README-USAGE.md [hunk 1]
```diff
@@ -1,28 +1,109 @@
-# How to use the Message Meet Widget
+# Spark Message and Meet Widget
 
-Our widget is built using React (https://github.com/facebook/react), Redux (https://github.com/reactjs/redux), and the Spark JS SDK (https://github.com/ciscospark/spark-js-sdk).
+The Spark Message and Meet widget allows developers to quickly and easily incorporate messaging through Cisco Spark into an application. This widget handles the heavy lifting of coordinating between your application and the Spark APIs, and provides a complete Spark UI and messaging experience without having to build all front end UI yourself.
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: I'm not sure I like the wording after the last "and". It's kinda intentionally *not* the complete experience.

File: packages/widget-message-meet/README-USAGE.md [hunk 2]
```diff
@@ -1,28 +1,109 @@
...
 
-Depending on how comfortable you are with these frameworks, there are are a number of ways you can use the widget on your website and in your code.
+Our widget is built using React <https://github.com/facebook/react>, Redux <https://github.com/reactjs/redux>, and the Spark JS SDK <https://github.com/ciscospark/spark-js-sdk>. This allows a developer to extend the UI to support different messaging contexts.
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: "Spark JavaScript SDK"

Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: maybe replace ". This allows" with ", allowing" and "to extend the" with "to extend their"

File: packages/widget-message-meet/README-USAGE.md [hunk 3]
```diff
@@ -1,28 +1,109 @@
...
+-   [Browser Support](#browser-support)
 
-The easiest way to get the Spark Message Meet Widget into your web site is to add the built resources and attach data attributes to your a container.
+## Features
 
-1. Follow these instructions to checkout and build the SDK (https://github.com/ciscospark/spark-js-sdk/blob/master/README.md)
-1. From the root of the sdk repo, run the following to build the widget:
-```sh
-PACKAGE=widget-message-meet npm run grunt:package -- build
+Currently, this widget supports:
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: No need for "Currently"

File: packages/widget-message-meet/README-USAGE.md [hunk 4]
```diff
@@ -1,28 +1,109 @@
...
+  spark:memberships_write
+  spark:messages_read
+  spark:messages_write
+  ```
+
+Head over to the Spark for Developers Documentation for more information about how to setup OAuth for your app: <https://developer.ciscospark.com/authentication.html>
+
+### CDN
+
+Using our CDN requires the least amount of work to get started. To get started, add the following into your HTML file:
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: You use "started" twice in four words.

File: packages/widget-message-meet/README-USAGE.md [hunk 5]
```diff
@@ -1,28 +1,109 @@
...
+```
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://code.s4d.io/widget-message-meet/main.css">
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: We should do something awful with `document.write` so this isn't required.

File: packages/widget-message-meet/README-USAGE.md [hunk 6]
```diff
@@ -1,28 +1,109 @@
...
-  data-user-id="sparky@ciscospark.com"
-  />
+
+### Build from Source
+
+1.  Follow these instructions to checkout and build the SDK <https://github.com/ciscospark/spark-js-sdk/blob/master/README.md>
+1.  From the root of the sdk repo, run the following to build the widget:
+
+  ```sh
+  PACKAGE=widget-chat npm run grunt:package -- build
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: Conflicts with @adamweeks's name changes?

File: packages/widget-message-meet/README-USAGE.md [hunk 7]
```diff
@@ -1,28 +1,109 @@
...
+1.  The built bundles are located at `package/widget-chat/dist`.
+
+## Usage
+
+### HTML
+
+The easiest way to get the Spark Chat Widget into your web site is to add the built resources and attach data attributes to your a container.
+
+1.  If you're using our CDN, skip to step 2.
+  -  Copy the resources in the `dist` directory to own project and add a `<script />` tag to your page to include the `bundle.js` and a `<link />` tag to include `main.css`.
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: wordy. consider multiple sentences/bullets

File: packages/widget-message-meet/README-USAGE.md [hunk 8]
```diff
@@ -1,28 +1,109 @@
...
+1.  Create a container where you which to include the chat widget and add the following attributes to configure the widget:
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: "where you which"?

File: packages/widget-message-meet/README-USAGE.md [hunk 9]
```diff
@@ -1,28 +1,109 @@
...
+  - `data-toggle="spark-chat"`: (required)
+  - `data-access-token`: Access token for the user account that is initiating the chat session. For testing purposes you can use a developer access token from <https://developers.ciscospark.com>
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: I think you can remove "that is"

File: packages/widget-message-meet/README-USAGE.md [hunk 10]
```diff
@@ -1,28 +1,109 @@
...
+  - `data-user-id`: User Id or email of the target user.
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: I don't like "target user". "other user"? "recipient"?

File: packages/widget-message-meet/README-USAGE.md [hunk 11]
```diff
@@ -1,28 +1,109 @@
...
+  <MessageMeetWidget accessToken="YOUR_ACCESS_TOKEN" userId="TARGET_USER_EMAIL" />,
+  document.getElementById('ELEMENT')
+);
 ```
+
+## Browser Support
+
+This widget supports the follow browsers:
+-   Current release of Chrome
+-   Current releaes of Firefox
```
Comment (ianwremmel) [inline packages/widget-message-meet/README-USAGE.md]: release is spelled wrong

File: packages/widget-message-meet/README-USAGE.md (hunk 5 above)
Comment (bzang) [inline packages/widget-message-meet/README-USAGE.md]: Nope. Too many chances for a flash of unstyled views that I'd much rather do this. The other option I would be to compile those class names and inject inline styles, but that would bloat the js bundle.

File: packages/widget-message-meet/README-USAGE.md (hunk 6 above)
Comment (bzang) [inline packages/widget-message-meet/README-USAGE.md]: Oops, missed that one.

File: packages/widget-message-meet/README-USAGE.md (hunk 11 above)
Comment (bzang) [inline packages/widget-message-meet/README-USAGE.md]: good catch

File: packages/widget-message-meet/README-USAGE.md (hunk 10 above)
Comment (bzang) [inline packages/widget-message-meet/README-USAGE.md]: I like `message recipient`

**Comment by bzang** [comment]
@ianwremmel Is there a way to prevent the document generator that's built in from overwriting the README.md in a project? I'd rather this be the main readme

**Pull request merged**